src/webapp/css/components/utilities.scss

```diff
@@ -6,7 +6,7 @@
 
 // Visibility modifiers
 .is-hidden {
-  display: none;
+  display: none !important;
```

Comment:
Is this just to make sure that it's taken into account no matter where you put it? Normally needing !important is an indicator my css is complicated and I can't figure out why something is happening so it makes me nervous seeing it.

Reply:
It's so that we can override whatever the display setting is. Unfortunately, where's I've been trying to use it, there are more specificity points (class + tag) than just this class :(.

My next change is a pretty large CSS refactor so hopefully I can fix this there.